content/sql/concepts/primary-keys/primary-keys.md

```diff
@@ -48,3 +48,33 @@ CREATE TABLE students (
 When the primary key for one table appears in a different table, it is called a foreign key.
 
 Why is this important? The most common types of joins will be joining a foreign key from one table with the primary key from another table. For instance, when we join the `orders` table and the `customers` table, we join on the `customer_id` column, which is a foreign key in `orders` and the primary key in `customers`.
+
+## Composite Keys
+
+Having one primary key per table is sometimes not enough to uniquely identify a row; in such cases we would use more than one column, or **composite keys**. This requirement should be detected during the designing phase of a database. 
+
+For example, a database containing vehicle parts will have to uniquely identify a row of parts. You could use either the `engine_ID` or  `body_ID`; however, this may create ambiguity as cars could get their engines swapped.
+
+Usually (but depending on local regulations) a car will require an engine part ID and a car body ID to be associated with a license plate. In this case you could have a table with more information about the car, such as `left_door_ID`, `gearbox_ID`, etc., but you would identify a specific car by two different aspects: its body and its engine. 
+
+A composite key would be useful in this case. This is how a `vehicle_registry` table would look (omitted extra parts/columns for brevity):
```

Comment:
```suggestion
For example, a database of car parts will have to uniquely identify a row of parts. You could use either the `engine_ID` or  `body_ID`; however, this may create ambiguity as cars could get their engines swapped.

Depending on local regulations, a car may require an engine part ID and a body ID to be associated with a license plate. One solution might be adding more row information about the car, such as `left_door_ID`, `gearbox_ID`, etc. But then a specific car would have to be identified by two different aspects: its body and its engine. 

A composite key would be useful in this case. This is how a `vehicle_registry` table might look (extra parts/columns omitted for brevity):
```